data/data.txt

```diff
@@ -0,0 +1,2 @@
+T | 1 | read book
```

Comment:
I'd suggest adding this to your gitignore

Reply:
Noted will add data storage to gitignore soon, thank you!